src/lib/pluginUtils.ts

```diff
@@ -16,6 +16,14 @@ import "moment";
 import * as moment from "moment-timezone";
 
 import { getArc } from "../arc";
+const WEB3 = require("web3");
+const web3 = new WEB3();
```

Comment:
how about moving this inside the function as local var ?

Reply:
Right.

Reply:
also .. `WEB3` -> `Web3`